lib/friendly_shipping/services/ups/shipping_methods.rb

```diff
@@ -80,11 +85,11 @@ def countries_by_code(code)
         ['PR', 'international', 'UPS Worldwide Express Plus', '54'],
         ['PR', 'international', 'UPS Worldwide Saver', '65'],
         ['DE', 'domestic', 'UPS Express 12:00 German domestic shipments', '74'],
-        ['OTHER', 'domestic','UPS Express', '07'],
-        ['OTHER', 'domestic','UPS Standard', '11'],
-        ['OTHER', 'international','UPS Worldwide Expedited', '08'],
-        ['OTHER', 'international','UPS Worldwide Express Plus', '54'],
-        ['OTHER', 'international','UPS Worldwide Saver', '65'],
+        ['OTHER', 'domestic', 'UPS Express', '07'],
+        ['OTHER', 'domestic', 'UPS Standard', '11'],
+        ['OTHER', 'international', 'UPS Worldwide Expedited', '08'],
+        ['OTHER', 'international', 'UPS Worldwide Express Plus', '54'],
+        ['OTHER', 'international', 'UPS Worldwide Saver', '65'],
         ['ALL', 'international', 'UPS Worldwide Express Freight', '96'],
         ['ALL', 'international', 'UPS Worldwide Express Freight Midday', '71']
       ].map do |origin_country_code, dom_or_intl, name, code|
```

Comment:
I think in a later PR this list of country mapping could be a frozen constant as well.

Reply:
Frozen in this PR